ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -290,6 +290,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getOwnerStatus = (ownerStatus: string): string => {
+      if (ownerStatus) {
+       if(ownerStatus=='PREVIOUS')
+       {
+         return 'HISTORICAL'
+       } 
+       else
+         return ownerStatus
+      }
+      else
+         return ''
```

Comment:
Minor syntax clean up, although not wrong, i would keep the `else` on the same line as the closing bracket. 
Also add brackets for the statement or keep the return on the same line. 

ie `} else return ownerStatus` or `} else { return ownerStatus }`
Some people prefer the brackets for consistency, i'm good with either.